editor/src/core/es-modules/package-manager/module-resolution.spec.ts

```diff
@@ -51,6 +80,18 @@ const sampleProjectContents: ProjectContentTreeRoot = contentsToTree({
     null,
     0,
   ),
+  '/src/dep-object.js': textFile(
+    textFileContents('export const DepObject = 1', unparsed, RevisionsState.CodeAhead),
```

Comment:
shouldn't this export an object? :)

Reply:
Nah this was just for testing the shape of `package.json`. The exported const here is ignored